### PR TITLE
For # 4927: Implement Settings Menu and Settings sub menus

### PR DESCRIPTION
**Pull request by mcarare**

### CODE REVIEW

**Comment by mcarare** [comment]
Note: Marked this as ready for review mainly so we can land strings and finish work that does not need any feedback. 

The commits messages describe what changes have been implemented.
Not the scope of this PR:
-  change the way the language list is displayed
-  change the way the cookies options are displayed
-  manage sites change (is it moved from  search? no intermediary screen for general settings)

File: app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -113,4 +113,10 @@
 
     <color name="settings_summary_text">#ffbbbcc5</color>
     <color name="settings_whats_new">#FFFF1AD9</color>
+
+    <!-- Preferences -->
+    <color name="preference_title_text">#FBFBFE</color>
+    <color name="preference_summary_text">#BFBFC9</color>
```
[the remark this answers is not in the document]
Comment (jonalmeida) [reply app/src/main/res/values/colors.xml:119]: You could possibly ignore this for now as we're fixing colour references when landing the nightly theme that's in a separate PR.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -276,10 +282,10 @@
 
     <!-- Three options for Cookie blocking. Yes (All cookies Blocked), Block 3rd-party cookies
     only (Only 3rd-party Cookies Blocked), No (No cookies blocked) -->
-    <string name="preference_privacy_should_block_cookies_no_option">No</string>
+    <string name="preference_privacy_should_block_cookies_no_option2">No thanks</string>
     <string name="preference_privacy_should_block_cookies_third_party_tracker_cookies_option">Block 3rd-party tracker cookies only</string>
     <string name="preference_privacy_should_block_cookies_third_party_only_option">Block 3rd-party cookies only</string>
-    <string name="preference_privacy_should_block_cookies_yes_option">Yes</string>
+    <string name="preference_privacy_should_block_cookies_yes_option2">Yes please</string>
```
Comment (jonalmeida) [inline app/src/main/res/values/strings.xml:288]: external-content duckduckgo com

**Review by jonalmeida — DISMISSED** [review]

**Comment by jonalmeida** [comment]
> Note: Marked this as ready for review mainly so we can land strings and finish work that does not need any feedback.

Landing. 👌

**Comment by pocmo** [comment]
> Landing. 👌

Mergify waits for approval here 😅

**Comment by mcarare** [comment]
> > Landing. 👌
> 
> Mergify waits for approval here 😅

@mergify mergify bot dismissed jonalmeida’s stale review via ff72322 5 hours ago
 He is doing this after every rebase.

**Review by ionutbedregeanu — APPROVED** [review]

**Comment by jonalmeida** [comment]
> > Landing. 👌
> 
> Mergify waits for approval here 😅

My bad. 🤦

**Pull request merged**